client/src/components/Overview/StyleSelector.jsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function StyleSelector() {
-  return null;
+function Style({ style, handleStyleSelect, selected }) {
+  // TODO: handle selected
+  return (
+    <div onClick={() => handleStyleSelect(style.style_id)}>
+      <img
+        src={style.photos[0].thumbnail_url}
+        alt={`${style.name} style thumbnail`}
+        width="50"
+      />
+    </div>
+  );
 }
 
+Style.propTypes = {
+  style: PropTypes.object.isRequired,
+  selected: PropTypes.bool.isRequired,
+  handleStyleSelect: PropTypes.func.isRequired,
+};
+
+function StyleSelector({ styles, selectedStyleId, handleStyleSelect }) {
+  // TODO: use color-thief to extract average color of thumbnails server-side
+  const stylesArray = styles.map((style) => (
```

Comment:
You can just directly return this and skip setting to a var then calling it later

Reply:
Yes good point. I think I had separated it in case I had more complicated logic in the return statement, but that ended up not being the case.